packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -144,6 +152,7 @@ export const JETPACK_PRODUCTS_LIST = <const>[
 	...JETPACK_VIDEOPRESS_PRODUCTS,
 	...JETPACK_SOCIAL_PRODUCTS,
 	...JETPACK_BACKUP_ADDON_PRODUCTS,
+	...JETPACK_SOCIAL_ADVANCED_PLANS,
```

Comment:
Very nitpicky, but let's put the lines with the Social products right below each other.